app/services/file/file-navigator/file-tree.model.spec.ts

```diff
@@ -15,6 +15,7 @@ function reprTree(tree: FileTreeStructure) {
 
 function reprNode(node: FileTreeNode, indent = 0) {
     const rows = [];
+    // tslint:disable-next-line:no-unused-variable
     for (const [_, child] of node.children) {
```

Comment:
the _ was failing the build, might have been a package update as this hasn't change for a long time.

Reply:
remove _ just `[,child]`